frontend/package-lock.json

```
@@ -12204,9 +12204,9 @@
       }
     },
     "node_modules/socket.io-parser": {
-      "version": "4.2.2",
-      "resolved": "https://registry.npmjs.org/socket.io-parser/-/socket.io-parser-4.2.2.tgz",
-      "integrity": "sha512-DJtziuKypFkMMHCm2uIshOYC7QaylbtzQwiMYDuCKy3OPkjLzu4B2vAhTlqipRHHzrI0NJeBAizTK7X+6m1jVw==",
+      "version": "4.2.3",
```

Me parece que la eneinaksakjshdjashdjajksnfajnncsasnncanscja jncjan 4.2.1. ajksdjkashdjhasjd

```suggestion
      "version": "4.2.1",
```